### ISSUE TITLE
Corana Warn App enthält jetzt auch Schlüssel anderer Europäischer Länder

### ISSUE THREAD

**Issue by nofaceinbook** [micb25/dka#20]
Hallo mic25,

sehe auf Deinen tollen Dashboards (vielen Dank!) seit letzter Woche einen größeren Anstieg insbesondere bei Personen die Diagnose Schlüssel teilen. Seit letzter Woche werden für die CWA ja auch Keys anderer europäischer Länder (bislang imho noch nicht viele) veröffentlicht. Deine Zahlen beziehen sich aber auf Deutschland. Falls möglich und noch nicht geschehen, solltest Du für Deine Statistiken nur die Deutschen Keys übernehmen.

Grüße

**Comment by felixlen** [comment]
Exactly. This can also be a source of confusion, as the `keyCount` in the log files of the exposure notification api in iOS start to diverge from the number of keys in the dashboard. The basic change necessary would be change `DE` to `EUR` in the URLs in `scripts/CWA_TEKs_download_and_process_hourly.py`. From my observation, the first date where the `keyCount` starts to diverge is `2020-10-27`.

**Comment by ndegendogo** [comment]
Maybe consider to have separate boards for DE and EUR.
DE numbers are better comparable with previous numbers
EUR numbers better match what I see on my device; and actually, it shows also the extended European coverage.

**Comment by ndegendogo** [comment]
@micb25 please consider to visualize the keys from the 'EUR' endpoint, as all cwa installations >= 1.5 are using these keys. I assume that number of cwa installations <= 1.3 are negligible.
Please note that also the 'DE' endpoint contains keys from the EGFS; [see more details here](https://github.com/corona-warn-app/cwa-documentation/issues/503).
